test_endpoints.http

```diff
@@ -729,6 +729,28 @@ content-type: application/json
 POST http://localhost:8011
 content-type: application/json
 
+{
+    "jsonrpc": "2.0",
+    "id": "1",
+    "method": "zks_getConfirmedTokens",
+    "params": [0, 100]
+}
+
+###
+POST http://localhost:8011
+content-type: application/json
+
+{
+    "jsonrpc": "2.0",
+    "id": "1",
+    "method": "zks_getAllAccountBalances",
+    "params": ["0x364d6D0333432C3Ac016Ca832fb8594A8cE43Ca6"]
```

Comment:
nit: Could we update this to be one of the Rich Wallet addresses? Just so we know it'll return something a balance `era_test_node run`